src/constraint_system/composer.rs

```diff
@@ -736,7 +791,8 @@ mod tests {
     }
 
     #[test]
-    // XXX: Move this to integration tests
+    #[ignore]
+    // XXX: Test works in principle, fails due to table size Vs number of lookups
```

Comment:
Make sure this is extracted into a separate issue so it doesn't get lost.

Reply:
okay